package.json

```diff
@@ -31,31 +31,34 @@
     "vuex": "^2.4.0"
   },
   "devDependencies": {
+    "autoprefixer": "^9.8.8",
```

Comment:
This was not an explicit dependency before, but looks like it was used - does it need to be here?